janusgraph-core/pom.xml

```diff
@@ -66,12 +66,10 @@
         <dependency>
             <groupId>commons-collections</groupId>
             <artifactId>commons-collections</artifactId>
-            <version>${commons.collections.version}</version>
         </dependency>
         <dependency>
             <groupId>commons-configuration</groupId>
             <artifactId>commons-configuration</artifactId>
-            <version>1.10</version>
```

Comment:
Please verify all the removed lines here are defined in the top level with consistent versions.